packages/components/tooltip/src/tooltip.ts

```diff
@@ -19,6 +19,11 @@ export const {
 } = createModelToggleComposable('visible' as const)
 
 export const useTooltipProps = buildProps({
+  ...usePopperProps,
+  ...useTooltipModelToggleProps,
+  ...useTooltipContentProps,
+  ...useTooltipTriggerProps,
+  ...usePopperArrowProps,
```

Comment:
Line#34

```ts
hideAfter: {
  type: Number,
  default: 200,
},
```

Maybe we could remove this prop, because it is contained in `useTooltipContentProps`.